dace/transformation/dataflow/local_storage.py

```diff
@@ -54,7 +54,9 @@ def apply(self, sdfg):
 
         array = self.array
         if array is None or len(array) == 0:
-            array = graph.edges_between(node_a, node_b)[0].data.data
+            array = next(e.data.data
+                         for e in graph.edges_between(node_a, node_b)
+                         if e.data.data is not None and e.data.wcr is None)
```

Comment:
Why are outputs with wcr excluded (assuming they have not been explicitly picked through the transformation parameters)?

Reply:
Nvm, we have accumulate transient now